crates/chain-config/src/snapshots/fuel_core_chain_config__config__tests__snapshot_simple_coin_state.snap

```diff
@@ -26,7 +26,7 @@ expression: json
     "max_gas_per_tx": 100000000,
     "max_script_length": 1048576,
     "max_script_data_length": 1048576,
-    "max_storage_slots": 255,
+    "max_storage_slots": 131072,
```

Comment:
we probably shouldn't be changing the max storage slots for the default config?

Reply:
Without it, a new e2e test will fail=) I can change it only for the integration test in the e2e module, but maybe it is also will be useful for benches in the future, so I decided to change the default values

Reply:
maybe we should make a new default for these kinds of benchmarking / integ tests that doesn't apply to general usage